runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -1747,6 +1757,12 @@ private void updateThreadMetrics() {
     maxOutstandingBundles.addValue(workUnitExecutor.maximumElementsOutstanding());
   }
 
+  private void sendWorkerMessage() throws IOException {
```

Comment:
Can you populate other processed signals here as well? Namely maximum_thread_count, active_bundle_count, maximum_bundle_count.

Reply:
Done.